src/redismodule.h

```diff
@@ -306,7 +306,6 @@ typedef enum {
 #define REDISMODULE_CMD_KEY_INSERT (1ULL<<6)
 #define REDISMODULE_CMD_KEY_DELETE (1ULL<<7)
 #define REDISMODULE_CMD_KEY_NOT_KEY (1ULL<<8)
-#define REDISMODULE_CMD_KEY_INCOMPLETE (1ULL<<9)
```

Comment:
besides the fact we're changing the reply of COMMAND (for MIGRATE), this change can cause a compilation error in modules.

perhaps we need to tag this PR with breaking-change?